tests/test_intake_solr.py

```diff
@@ -9,7 +9,7 @@
 from .util import start_solr, stop_docker, TEST_CORE
 
 CONNECT = {'host': 'localhost', 'port': 9200}
-TEST_DATA_DIR = 'tests'
+TEST_DATA_DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I was having issues running the test from within PyCharm, this fixed the problem for me.

Reply:
seems good practice - should not require a particular CWD